### ISSUE TITLE
Unable to sort pending review table by Creation time

### ISSUE THREAD

**Issue by fcrozat** [openSUSE/cavil#68]
It looks like some recent update / rewrite in UI dropped the ability to sort the pending review table using the different columns.

I'm not longer able to sort it by Creation time

**Comment by kraih** [comment]
That's correct, the old table implementation was client side, and got too slow with the amount of data we have to handle these days. The new, faster implementation, will require sorting to be done server side on SQL level.

**Comment by fcrozat** [comment]
We might not need to allow columns sorting but in that case, the default should still be "creation" time (the current sorting is a bit strange, having "obs leap" at the top)

**Comment by kraih** [comment]
The current default sorting is based on priority, which is important for reviewers.

**Comment by kraih** [comment]
If sorting is particularly important for any existing workflows, please let me know, and i'll prioritise this feature accordingly.

**Comment by fcrozat** [comment]
Well, if we can get rid of the obs#leap one, we don't need custom sorting.
